frontend/src/components/common/ChipGroup/LanguageChipGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import { WhatsAppLanguages } from '@shared/clients/whatsapp-client.class/types'
+import { capitalize } from 'lodash'
+import { Dispatch, SetStateAction } from 'react'
+
+import { Chip } from './Chip'
+
+interface LanguageChipGroupProps {
+  selected: WhatsAppLanguages
+  setSelection: Dispatch<SetStateAction<WhatsAppLanguages>>
+}
+
+const languageChipGroupStyle = {
+  display: 'flex',
+  gap: '1rem',
+}
+
+export const LanguageChipGroup = ({
+  selected,
+  setSelection,
+}: LanguageChipGroupProps) => {
+  return (
+    <div style={languageChipGroupStyle}>
+      {Object.keys(WhatsAppLanguages).map((language) => {
+        const value =
+          WhatsAppLanguages[language as keyof typeof WhatsAppLanguages]
```

Comment:
If we use `Object.entries()`, would we be able to avoid this access pattern?

Reply:
Update: This became a non-issue due to #2161